fawltydeps/settings.py

```diff
@@ -325,6 +326,16 @@ def populate_parser_options(parser: argparse._ActionsContainer) -> None:
             " to looking for supported files in the current directory)"
         ),
     )
+    parser.add_argument(
+        "--venv",
+        type=Path,
+        metavar="VENV_DIR",
+        help=(
+            "Where to find a virtualenv that has the project dependencies"
```

Comment:
"where to find" makes it seem like we will go looking for the venv while the reality is that we expect the user to provide the path.
```suggestion
            "Path to a virtualenv that has the project dependencies"
```